source/xlsxreader.d

```diff
@@ -90,20 +207,20 @@ struct Sheet {
 		return app.data;
 	}
 
-	void printTable() {
+	void printTable() const {
 		writeln(this.toString());
 	}
 
 	// Column
 
-	Iterator!T getColumn(T)(size_t col, size_t start, size_t end) {
-		auto c = this.iterateColumn!T(col, start, end);
-		return Iterator!T(c.array);
+	Iterator!T getColumn(T)(size_t col, size_t startRow = 0, size_t endRow = size_t.max) {
+		auto c = this.iterateColumn!T(col, startRow, endRow);
+		return typeof(return)(c.array); // TODO: why .array?
```

Comment:
Because it makes the Iterator type a lot easier to use.

Reply:
Resolved via 429eab3.